web_external/js/MinervaModel.js

```diff
@@ -9,6 +9,14 @@ minerva.models.MinervaModel = girder.models.ItemModel.extend({
         }
     },
 
+    mM: function (minervaMetadata) {
+        if (minervaMetadata) {
```

Comment:
My motivation was that I got sick of typing out getMinervaMetadata/setMinervaMetadata all the time.  This is something that will be called often throughout the application.

Is there some other shorthand you can live with?  

`meta`?
`metadata`?
`mMetadata`?
`minerva`?

I'm willing to go with `minervaMetadata` if you can't stomach anything else.  :mask: